chart/inlets-operator/README.md

```diff
@@ -26,6 +26,15 @@
     kubectl apply -f ./artifacts/crd.yaml
     ```
 
+## Configuration
+
+The following table lists the configurable parameters of the `inlets-operator` chart and their default values,
+and can be overwritten via the helm `--set` flag.
+
+Parameter | Description | Default
+---                                           | ---                                                                   | ---
+`annotated-only`                              | if the operator should create a tunnel for un annotated services      | `false`
```

Comment:
Strictly, if we use this table we should add the other flags and parameters.

Would you mind doing that in this PR please?

Reply:
no problem...